### PR TITLE
Adding _app.js and _document.js as mentioned in Next.js documentation. 

### LINKED ISSUE #10: Add _app.js file in /pages
1. Add `_app.js` file in `/pages`. `_app.js` allows us to add global styles and components to our project. 
2. Change the way SCSS is imported. The SCSS import pattern should follow [this.](https://nextjs.org/docs/basic-features/built-in-css-support)
3. Add _document.js file in `/pages` directory. Read [this](https://nextjs.org/docs/advanced-features/custom-document)

### PR DESCRIPTION
**Pull request by invalidtoken**
Adding 
1. _app.js file in `/pages`
2. _document.js file in `/pages`
3. Using SASS modules as mentioned [here](https://nextjs.org/docs/advanced-features/custom-document)
4. Added SASS Guide in README.md

This PR is fixes #10

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = {
   ],
   rules: {
     'prettier/prettier': ['error', {}, { usePrettierrc: true }],
-    'react/react-in-jsx-scope': 'off'
+    'react/react-in-jsx-scope': 'off',
+    'react/prop-types': 'off'
```
Comment (ankushdharkar) [inline .eslintrc.js:29]: Why disable the prop-types?

Comment (invalidtoken) [reply .eslintrc.js:29]: Because we will have to define the prop-type for every single component. 
Should I remove this?

**Comment by invalidtoken** [comment]
@ankushdharkar I have removed `react/prop-types: "false"` flag. I have also added the prop-types module.

File: .eslintrc.js (hunk 1 above)
Comment (ankushdharkar) [reply .eslintrc.js]: We should always define them, right? So we know what the component is expecting, @invalidtoken?

File: pages/_app.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const MyApp = (props) => {
+  const { Component, pageProps } = props;
+  return (
+    <div>
+      <Component {...pageProps} />
+    </div>
+  );
+};
+
+MyApp.propTypes = {
```
[the remark this answers is not in the document]
Comment (invalidtoken) [reply pages/_app.js:13]: Added `isRequired` for Component. This is definitely a good practice, thanks @shihabus for pointing out.

**Comment by invalidtoken** [comment]
@ankushdharkar @shihabus can you please re-review this PR.

File: pages/_app.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (shihabus) [reply pages/_app.js:13]: Seems fine

File: .eslintrc.js (hunk 1 above)
Comment (invalidtoken) [reply .eslintrc.js]: Yes, @ankushdharkar definitely.

**Pull request merged**